### ISSUE TITLE
Doesn't work after update to 22.03.0

### ISSUE THREAD

**Issue by definitio** [awilliams/wifi-presence#9]
```
Mon Sep 12 22:21:02 2022 daemon.err wifi-presence[3272]: Error: unable to connect to hostapd control socket "/var/run/hostapd/wlan0": ping error: read error from "PING" command: read unixgram /tmp/wp.wlan0->/var/run/hostapd/wlan0: i/o timeout
```

**Comment by awilliams** [comment]
Which version of `wifi-presence` are you using? This error can happen when the `/usr/bin/wifi-presence` binary is executed as a different user than the `/var/run/hostapd/wlan0` file. 

Please check that you're using the latest version, specifically that you have this section in your `/etc/init.d/wifi-presence` script: https://github.com/openwrt/packages/blob/992fcd1bd8770bb44a56bc4173ac3befe0fa16ef/net/wifi-presence/files/wifi-presence.init#L64-L75

That section was added in #6 to address the very issue I believe you're experiencing (described in #3). It's included in version `0.1.2 release 2`.

I've tested on two different routers both running 22.03.0 and everything works as excepted. Please report back if that doesn't work.

**Comment by definitio** [comment]
I used old version from your repo: 0.1.1-1. v0.2.0 works. Thanks.
